src/main/java/com/limechain/runtime/RuntimeEndpoint.java

```diff
@@ -0,0 +1,20 @@
+package com.limechain.runtime;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+/**
+ * Holds all Runtime API endpoints' names.
+ */
+@Getter
+@AllArgsConstructor
+// TODO: Define all endpoints
```

Comment:
I could do this now, but since I'm not yet sure we're sticking to this design decision, I thought I'd wait on discussions.

Anyway, this is the class to incrementally add new endpoint definitions to. In this sense, it's not yet finished.

Reply:
I'm fine with this approach. However, I see some of the endpoints have parameters related to them. It could be a good idea to add a string field in this enum which would serve as a description of the endpoint/of the parameters required.

Reply:
I'm not sure we'd want a string field, since this info is developer-relevant only and resides in the spec anyways. In this sense, we have no use for descriptions being available in the java runtime; I see Javadocs rather fit. 
However, on the matter of the different parameters for each endpoint, I see `Runtime` as the place for constructing methods nicely encapsulating this interface (the runtime API) with methods, which will have proper javadocs (see `Runtime::executeBlock` for example). So the `RuntimeEndpoint` serves as nothing more than an exhaustive identifier for all existing function names (and a type to identify an `endpoint` by), but without holding the entire signature for the API. 

I am, however, open for better naming...

Reply:
I have no suggestions for better naming. I'm marking the thread as resolved. +1 for javadocs